projects/storefrontapp-e2e-cypress/cypress/integration/regression/password-visibility/password-visibility.core-e2e-spec.ts

```diff
@@ -0,0 +1,58 @@
+import { viewportContext } from '../../../helpers/viewport-context';
+import { standardUser } from '../../../sample-data/shared-users';
+import { generateMail, randomString } from '../../../helpers/user';
+import { clearAllStorage } from '../../../support/utils/clear-all-storage';
+import { waitForPage } from '../../../helpers/checkout-flow';
+
+context('Password Visibility', () => {
+  viewportContext(['mobile', 'desktop'], () => {
+    beforeEach(() => {
+      cy.window().then((win) => {
+        win.sessionStorage.clear();
```

Comment:
we can remove 
```
cy.window().then((window) => { 
win.sessionStorage.clear();
});
```

since clearAllStorage includes that already

Reply:
Done